instrumentation/spring/spring-boot-autoconfigure-3/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/web/RestClientInstrumentationAutoConfiguration.java

```diff
@@ -25,10 +22,8 @@
  *
  * <p>Adds Open Telemetry instrumentation to {@link RestClient} beans after initialization
  */
-@ConditionalOnBean(OpenTelemetry.class)
-@ConditionalOnProperty(name = "otel.instrumentation.spring-web.enabled", matchIfMissing = true)
+@SpringBootInstrumentation(module = "spring-web")
```

Comment:
Maybe easier to understand:

```suggestion
@ConditionalOnOpenTelemetry(module = "spring-web")
```

Or

```suggestion
@ConditionalOnOpenTelemetryInstrumentation(module = "spring-web")
```

Reply:
good idea